Source/Core/Common/FileUtil.cpp

```diff
@@ -885,10 +885,8 @@ std::string GetExeDirectory()
 #endif
 }
 
-std::string GetSysDirectory()
+std::string CreateSysDirectoryPath()
```

Comment:
`CreateSysDirectoryPath` should probably be `static` since it's not in the header (though since it's in the `File` namespace this isn't as important as if it were in the global namespace).

Reply:
Done.